src/main/java/frc/robot/commands/complex/SpeakerAuto.java

```diff
@@ -11,7 +13,7 @@ public class SpeakerAuto extends SequentialCommandGroup {
     public SpeakerAuto() {
         addCommands(
                 new ShooterPivotPosTolerance(ArmPositions.SPEAKER_SHOT),
-                new ShooterFlywheelSpeedTolerance(ShooterSpeeds.SPEAKER),
+                new ShooterFlywheelSpeedToleranceMeters(ShooterSpeeds.SPEAKER),
```

Comment:
the name of this command seems to imply it is a constant; I'd call it something else but I understand that it may mean something specific that I do not have context for